pkg/mcs/scheduling/server/apis/v1/api.go

```diff
@@ -120,12 +119,15 @@ func NewService(srv *scheserver.Service) *Service {
 func (s *Service) RegisterSchedulersRouter() {
 	router := s.root.Group("schedulers")
 	router.GET("", getSchedulers)
+	router.GET("/diagnostic/:name", getDiagnosticResult)
```

Comment:
We don't need it for now.

Reply:
why? It can pass tests

Reply:
This feature is not widely used, to reduce the problem, I think we can hold it instead of expose it.

Reply:
I think it's best to keep forwarding, one reason is that it will make matching rules simple, and another reason is to avoid us missing an interface or creating incompatibility issues